src/frontend/utils/conversejs/converse-plugins/chatPlugin.ts

```diff
@@ -15,14 +17,16 @@ enum StanzaType {
 }
 
 enum StanzaMessageType {
-  CLASSIC_MESSAGE = 'CLASSIC_MESSAGE',
+  LIVE_MESSAGE = 'LIVE_MESSAGE',
   HISTORY = 'HISTORY',
   SUBJECT = 'SUBJECT',
   UNRECOGNIZED = 'UNRECOGNIZED',
 }
 
 const PLUGIN_NAME = 'chat-plugin';
 
+const ANONYMOUS_ID_PREFIX = 'anonymous';
```

Comment:
why are you not using `ANONYMOUS_ID_PREFIX` from `default/chat.ts` ?

Reply:
I missed it, ty.
I also noticed I forgot to change it in converse.spec.tsx (so I also changed it there)